pyanaconda/modules/security/certificates/installation.py

```diff
@@ -38,6 +41,24 @@ def __init__(self, sysroot, certificates):
     def name(self):
         return "Import CA certificates"
 
+    def _dump_certificate(self, cert, root):
+        """Dump the certificate into specified file and directory."""
+        dst_dir = join_paths(root, cert.dir)
+        if not os.path.exists(dst_dir):
+            log.debug("Path %s for certificate %s does not exist, creating.",
+                      dst_dir, cert.filename)
+            mkdirChain(dst_dir)
+
+        dst = join_paths(dst_dir, cert.filename)
+        with open(dst, 'w') as f:
```

Comment:
Maybe we should specify the text mode explicitly ? Eq. `wt` to make it apparent that we are writing text data right now.

Reply:
I'd suggest this as upstream patch first.